stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -98,102 +106,180 @@ public Stripe(String publishableKey) throws AuthenticationException {
-    }
 
+    /**
+     * Retrieve a token for inspection on a specific {@link Executor}, using the publishable key
+     * that was used to create the token.
+     *
+     * @param tokenId the id of the token being requested
+     * @param publishableKey the key used to create the token
+     * @param executor an {@link Executor} on which to run this operation, or {@code null} to run
+     *                 on a default background executor
+     * @param callback a {@link TokenCallback} to receive the result
+     */
     public void requestToken(
-            final String tokenId,
-            final String publishableKey,
-            final Executor executor,
-            final TokenCallback callback) {
+            @NonNull final String tokenId,
+            @NonNull @Size(min = 1) final String publishableKey,
+            @Nullable final Executor executor,
+            @NonNull final TokenCallback callback) {
         try {
-            if (tokenId == null)
-                throw new RuntimeException("Required Parameter: 'tokenId' is required to request a token");
+            if (tokenId == null) {
+                throw new RuntimeException("Required Parameter: 'tokenId' " +
+                        "is required to request a token");
+            }
 
-            if (callback == null)
-                throw new RuntimeException("Required Parameter: 'callback' is required to use the requested token and handle errors");
+            if (callback == null) {
+                throw new RuntimeException("Required Parameter: 'callback' " +
+                        "is required to use the requested token and handle errors");
+            }
 
             validateKey(publishableKey);
 
             tokenRequester.request(tokenId, publishableKey, executor, callback);
-        }
-        catch (AuthenticationException e) {
+        } catch (AuthenticationException e) {
             callback.onError(e);
         }
     }
 
-    public void createToken(
-            final Card card,
-            final String publishableKey,
-            final Executor executor,
-            final TokenCallback callback) {
-        try {
-            if (card == null) {
-                throw new RuntimeException(
-                        "Required Parameter: 'card' is required to create a token");
-            }
-
-            if (callback == null) {
-                throw new RuntimeException(
-                        "Required Parameter: 'callback' is required to use the created " +
-                                "token and handle errors");
-            }
-
-            validateKey(publishableKey);
+    /**
+     * Set the default publishable key to use with this {@link Stripe} instance.
+     *
+     * @param publishableKey the key to be set
+     * @throws AuthenticationException if the key is null, empty, or a secret key
+     */
+    public void setDefaultPublishableKey(@NonNull @Size(min = 1) String publishableKey)
```

Comment:
I also reordered the methods to put them in public/package/private order, so some of these that look like "brand new method" in github are just copy-paste operations. Plus javadoc.